src/sqltrie/serialized.py

```diff
@@ -88,8 +88,10 @@ def ls(self, key, with_values=False):
             yield from entries
 
     def traverse(self, node_factory, prefix=None):
-        def _node_factory_wrapper(path_conv, path, children, value):
-            return node_factory(path_conv, path, children, self._load(path, value))
+        def _node_factory_wrapper(path_conv, path, children, *value):
+            return node_factory(
+                path_conv, path, children, self._load(path, value[0] if value else None)
+            )
```

Comment:
node factory value is defined as an optional positional arg in pygtrie (since there is a difference between a node having no value set and a node value being set to `None`)